pkg/deploy/api/validation/validation.go

```diff
@@ -255,29 +257,18 @@ func validateImageChangeParams(params *deployapi.DeploymentTriggerImageChangePar
 	errs := fielderrors.ValidationErrorList{}
 
 	if len(params.From.Name) != 0 {
-		if len(params.From.Kind) == 0 {
-			params.From.Kind = "ImageStream"
-		}
 		kinds := sets.NewString("ImageRepository", "ImageStream", "ImageStreamTag")
 		if !kinds.Has(params.From.Kind) {
```

Comment:
I was doing that but it breaks current DCs with ImageRepository or ImageStream set in their ICTs.